apps/ima/translations/src/en/fields.json

```diff
@@ -17,5 +17,47 @@
         "hint": "You cannot give immigration advice. You must answer all the questions as if you are the person named on the duty to remove letter."
       }
     }
+  },
+  "does-exception-apply":{
+    "legend": " ",
```

Comment:
why is this empty? You can use` legend: {
className: 'visually hidden'}` if you need to instead of making it an empty string